ts/components/ui/Markdown/rules/heading.ts

```diff
@@ -0,0 +1,40 @@
+import { H1, H2, H3 } from "native-base";
+import * as React from "react";
+import { ReactOutput, SingleASTNode, State } from "simple-markdown";
+
+import { makeReactNativeRule } from ".";
+
+function rule() {
+  return (
+    node: SingleASTNode,
+    output: ReactOutput,
+    state: State
+  ): React.ReactNode => {
+    // tslint:disable-next-line no-let
+    let ComponentType = null;
```

Comment:
What about, in order to avoid bending the lint rule, something like
```typescript
const componentTypes = { [ key: number ]: React.Component } = {
  1: H1, 
  2: H2,
  3: H3
};
const ComponentType = node.level in componentTypes ? componentTypes[node.level] : null;
```
? 
While the `componentTypes` prototype may be altered (thus technically requiring an `hasOwnProperty()` check on top of the `in` one) I think that realistically this is a pretty self-contained case, so probably that additional check would not be needed. 

Also, is there any particular reason why the name is in CamelCase?